src/multio/action/scale/Scaling.cc

```diff
@@ -0,0 +1,38 @@
+#include "Scaling.h"
+
+#include <cstdlib>
+#include <string>
+
+#include "multio/LibMultio.h"
+#include "multio/util/Substitution.h"
+
+
+namespace multio::action {
+
+
+ScaleScaling::ScaleScaling(const config::ComponentConfiguration& compConf) : hasScaling_(false), scaleFactor_{} {
+
+    const auto mappings = compConf.parsedConfig().has("mapping-definition")
```

Comment:
There seems to be some duplication of logic here as well. Can we remove the one in `Scale.cc`?

Reply:
I was weighing that option: I can either create individual plans for fields where scaling is needed or as I currently have add it to plans where only a subset of fields actually need scaling specified by case in the plan. BUT then I wanted to have a check to leave this action as early as possible. I can move the logic in a util function and call it, but this would still be in all three cases. Or I can remove the whole logic of checking as soon as possible. Or I can retrieve the param only in scale.cc and then pass it to the apply function (But this I didn't prefer since I wanted to have the Mapping and scaling as enclosed as possbile sind I pass the metadata anyway and hence have the information anyway)